deepchem/models/torch_models/cgcnn.py

```diff
@@ -247,6 +247,29 @@ def forward(self, dgl_graph):
       return proba, logits
 
 
+def pooling(graph, feat):
+  seglen = tuple(graph.batch_num_nodes(None).detach().numpy())
+  value = torch.split(graph.ndata[feat], seglen)
+  return torch.stack([torch.mean(inp, 0) for inp in value])
+
+class CGCNN_OV(CGCNN):
+  """Version of Crystal Graph Convolutional Neural Network (CGCNN) adapted for conversion to IR."""
+
+  def __init__(self, in_node_dim, hidden_node_dim, in_edge_dim,
+                num_conv, predictor_hidden_feats, n_tasks, mode, n_classes):
+    super().__init__(in_node_dim, hidden_node_dim, in_edge_dim,
+                      num_conv, predictor_hidden_feats, n_tasks, mode, n_classes)
```

Comment:
Please run Python code style locally before push (to make CI happy: https://github.com/dkurt/deepchem/pull/7/checks?check_run_id=2229127007)